CMakeLists.txt

```
@@ -179,30 +180,57 @@ check_library_exists(rt shm_open "" LMMS_HAVE_LIBRT)
 
 LIST(APPEND CMAKE_PREFIX_PATH "${CMAKE_INSTALL_PREFIX}")
 
-FIND_PACKAGE(Qt5 5.9.0 COMPONENTS Core Gui Widgets Xml REQUIRED)
-FIND_PACKAGE(Qt5 COMPONENTS LinguistTools QUIET)
+IF(WANT_QT6)
+	MESSAGE("Building with Qt6 is experimental, not guaranteed to succeed")
+	SET(QT_VERSION_MAJOR 6)
+	SET(WANT_VST OFF)
+	SET(STATUS_VST "VSTs not supported with Qt 6")
+	SET(STATUS_QT6 "ENABLED")
+ELSE()
+	SET(QT_VERSION_MAJOR 5)
+	SET(STATUS_QT6 "NOT ENABLED")
+ENDIF()
+
+FIND_PACKAGE(Qt${QT_VERSION_MAJOR} COMPONENTS Core Gui Widgets Xml REQUIRED)
```

```suggestion
IF(WANT_QT6)
	MESSAGE("Building with Qt6 is experimental, not guaranteed to succeed")
	SET(QT_VERSION_MAJOR 6)
	SET(LMMS_QT_MIN_VERSION 6.0.0)
	SET(WANT_VST OFF)
	SET(STATUS_VST "VSTs not supported with Qt 6")
	SET(STATUS_QT6 "ENABLED")
ELSE()
	SET(QT_VERSION_MAJOR 5)
	SET(LMMS_QT_MIN_VERSION 5.9.0)
	SET(STATUS_QT6 "NOT ENABLED")
ENDIF()

FIND_PACKAGE(Qt${QT_VERSION_MAJOR} ${LMMS_QT_MIN_VERSION} COMPONENTS Core Gui Widgets Xml REQUIRED)
```
@Rossmaxx Here's one way to fix the Qt minimum version requirement. We might want Qt6 to have a different minimum version than 6.0.0 though.